src/main/java/kg/jarkyn/server/Request/ParsedRequest.java

```diff
@@ -0,0 +1,19 @@
+package kg.jarkyn.server.Request;
+
+public class ParsedRequest {
+    private final String verb;
+    private final String path;
+
+    public ParsedRequest(String verb, String path) {
```

Comment:
yup, was thinking that too, but didn't put in till the knowledge about the other methods/verbs was introduced into the code